pixi.toml

```diff
@@ -196,6 +196,9 @@ deploy-latest = "mike deploy --push --update-aliases $RELEASE_VERSION latest"
 docs = { cmd = "mkdocs serve", description = "Serve the docs locally" }
 mike-serve = "mike serve"
 
+[feature.docs.pypi-dependencies]
+mkdocs-llmstxt = ">=0.3.0, <0.4"
```

Comment:
Id rather add this from conda-forge, I started adding it https://github.com/conda-forge/staged-recipes/pull/30696 .

Reply:
Thank you @baszalmstra for adding the conda package for this!